util/src/keys/directory.rs

```diff
@@ -474,6 +475,12 @@ pub fn restrict_permissions_owner(file_path: &Path) -> Result<(), i32>  {
 	}
 }
 
+#[cfg(windows)]
+pub fn restrict_permissions_owner(_file_path: &Path) -> Result<(), i32>  {
+	//TODO: implement me
+	Ok(())
```

Comment:
don't thinks so
windows restricts permissions to the user home directory on it's own